components/datatypes/impressions.schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/impressions",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Impressions",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:industry": [
+    "retail",
+    "financial_services",
+    "travel_and_hospitality",
+    "telecom",
+    "automotive",
+    "education",
+    "health_and_life_sciences",
+    "high_tech",
+    "manufacturing",
+    "media_and_entertainment",
+    "public_sector"
+  ],
+  "description": "Tracks when impressions are presented on a site, along with names, types, and counts.",
+  "definitions": {
+    "impression": {
+      "properties": {
+        "xdm:impressionDisplays": {
+          "title": "Impression Displays",
+          "$ref": "https://ns.adobe.com/xdm/datatypes/data/measure",
+          "description": "Counts the number of times an impression item has been displayed to a visitor."
+        },
+        "xdm:impressionID": {
+          "title": "Impression ID",
+          "type": "string",
+          "description": "Tracks the ID of an item in an impression array."
+        },
+        "xdm:impressionType": {
+          "title": "Impression Type",
+          "type": "string",
+          "description": "Tracks the type of impression in an impression array."
+        }
+      }
+    }
+  },
+  "meta:status": "experimental"
+}
```

Comment:
If there is a definition, you need to have an AllOF section

Reply:
I'll update that globally